scipy/optimize/optimize.py

```diff
@@ -2549,13 +2611,14 @@ def fmin_powell(func, x0, args=(), xtol=1e-4, ftol=1e-4, maxiter=None,
             return res['x']
 
 
-def _minimize_powell(func, x0, args=(), callback=None,
+def _minimize_powell(func, x0, args=(), bounds=None, callback=None,
                      xtol=1e-4, ftol=1e-4, maxiter=None, maxfev=None,
                      disp=False, direc=None, return_all=False,
                      **unknown_options):
     """
     Minimization of scalar function of one or more variables using the
-    modified Powell algorithm.
+    modified Powell algorithm. Note that bounds will be respected at
+    every iteration.
```

Comment:
`bounds` needs to be added to the docstring here. `bounds` should comply with those provided by `standardize_bounds`.

Reply:
Done.